lesson_l18/test_hw_l18_iterator.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import pathlib
+
+from assertpy import assert_that
+from logging import config
+
+from lesson_l18.decorators import setup_teardown_decorator
+
+path_to_project = pathlib.Path(f"{pathlib.Path.home()}/Projects/AQA_course/")
+
+logging.config.fileConfig(f"{path_to_project}/logging_config.ini")
+logging.getLogger('sampleLogger')
+
+def revers_list(execute_list):
```

Comment:
For what this method if you do not use it anywhere in proper way?

Reply:
I use the method revers_list for creating the expected result in test case test_revers_list_by_iterator.
![image](https://github.com/user-attachments/assets/118f628c-f0b0-4dd5-b944-3ab76045c818)